op-node/rollup/derive/deposits.go

```diff
@@ -21,26 +22,30 @@ func UserDeposits(receipts []*types.Receipt, depositContractAddr common.Address)
 			if log.Address == depositContractAddr && len(log.Topics) > 0 && log.Topics[0] == DepositEventABIHash {
 				dep, err := UnmarshalDepositLogEvent(log)
 				if err != nil {
-					errs = append(errs, fmt.Errorf("malformatted L1 deposit log in receipt %d, log %d: %w", i, j, err))
+					result = multierror.Append(result, fmt.Errorf("malformatted L1 deposit log in receipt %d, log %d: %w", i, j, err))
```

Comment:
Do we need multierror here when we can use native go error wrapping?
`err = fmt.Errorf("%v, %v", result, err)`? Unsure if it just works if `result` is `nil` (the first error)

Reply:
I mean you could chain errors natively. I kinda like multi-err b/c this was previously an array of errors.